tests/test_morphology.cpp

```diff
@@ -20,3 +20,7 @@ TEST_CASE("LoadNeurolucidaMorphology", "[morphology]") {
 
     REQUIRE(m.diameters().size() == 14);
 }
```

Comment:
Could you add a real test case where the new  constructor `Morphology(const HighFive::Group& group, unsigned int options = NO_MODIFIER)` is used